source/tool/chipsec/helper/oshelper.py

```diff
@@ -108,32 +119,39 @@ def loadHelpers(self):
             except:
                 pass
 
-    def __del__(self):
-        try:
-            destroy()
-        except NameError:
-            pass
-
-    def start(self, start_driver):
+    def start(self, start_driver, driver_exists=False):
         try:
             self.helper.create(start_driver)
-            self.helper.start(start_driver)
+            self.helper.start(start_driver, driver_exists)
         except (None,Exception) , msg:
             if logger().VERBOSE: logger().log_bad(traceback.format_exc())
             error_no = errno.ENXIO
             if hasattr(msg,'errorcode'):
                 error_no = msg.errorcode
             raise OsHelperError("Could not start the OS Helper, are you running as Admin/root?\n           Message: \"%s\"" % msg,error_no)
 
-    def stop( self ):
-        self.helper.stop()
-
-    def destroy( self ):
-        self.helper.delete()
+    def stop( self, start_driver ):
+        self.helper.stop( start_driver )
+        #self.helper.delete( start_driver )
 
     def is_driver_loaded(self):
         return self.helper.driver_loaded
 
+    #
+    # use_native_api
+    # Defines if CHIPSEC should use its own API or native environment/OS API
+    #
+    # Returns:
+    #   True  if CHIPSEC needs to use native OS/environment API
+    #   False if CHIPSEC needs to use its own (OS agnostic) implementation of helper API
+    #
+    # Currently, CHIPSEC will use native API only if CHIPSEC driver wasn't loaded
+    # (e.g. when --no_driver command-line option is specified).
+    # In future, it can can more conditions
+    # 
+    def use_native_api(self):

```

Comment:
This is literally the opposite of the `is_driver_loaded` function. Could you remove the `is_driver_loaded` function and replace its calls with `use_native_api`?